README.md

```
@@ -1,36 +1,173 @@
-First, run the development server:
+### 💌 모바일 초대장 서비스 Invia
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
+[<img src="https://img.shields.io/badge/-invia.co.kr-9c8e7e?style=flat&logo=google-chrome&logoColor=white" />](https://invia.co.kr) [<img src="https://img.shields.io/badge/-GitHub-black?style=flat&logo=github&logoColor=white" />](https://github.com/Bread-Barbershop/frontend) [<img src="https://img.shields.io/badge/release-v1.1.2-pink?style=flat&logo=google-chrome&logoColor=white" />]()
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)